src/graph_compute/bsp_scheduling.h

```diff
@@ -14,14 +14,18 @@ static void init_scheduling(vertex_t * const nodes, const vid_t cntNodes,
   // no-op
 }
 
-static void execute_round(const int round, vertex_t * const nodes, const vid_t cntNodes,
+static void execute_round(const int numRounds, vertex_t * const nodes,
```

Comment:
I don't understand - do you want me to check out pr_rollback, make this change and push it to advance pr_rollback?  I have a branch, specialized_vertex_t, that started off as the current pr_rollback and has evolved significantly since then.  The comments you are making here are quite relevant to this new branch, which is why I'm making the changes there, rather than pr_rollback.  I was expecting that after addressing your concerns I would do a pull request vs pr_rollback.  How should this process work?